src/site/topic6.rst

```diff
@@ -86,7 +86,7 @@ Implementation
     you are saying that you are implementing the interface, Java is expecting you to implement all abstract methods from
     the interface.
 
-        .. image:: img/warning_implement.png
+        .. image:: ../img/warning_implement.png
```

Comment:
img is a dead link, see [build](https://github.com/jameshughes89/cs102/pull/34/checks#step:5:198)